<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainControllerGUI</class>
 <widget class="QMainWindow" name="TrainControllerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainControllerGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="incSpeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Increase Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decSpeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease Speed</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentSetpoint">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="doorButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open/Close Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lightButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lights On/Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serviceBrake">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Brake</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string> Setpoint Speed</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentPower">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="powerLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="lightStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Status: OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="doorStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Staus: CLOSED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic/Manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeStatus">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode: Manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="eBrakeStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Brake Status: OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eBrakeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Brake</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="kpTextbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="kiTextbox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Ki</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
